tstrait/simulate_effect_size.py

```diff
@@ -190,7 +178,11 @@ def sim_trait(ts, model, *, num_causal=None, alpha=None, random_seed=None):
     model : tstrait.TraitModel
         Trait model that will be used to simulate effect sizes.
     num_causal : int, default None
-        Number of causal sites. If None, number of causal sites will be 1.
+        Number of causal sites that will be randomly selected . If both `num_causal` and
+        `causal_sites` are None, number of causal sites will be 1.
+    causal_sites : list, default None
+        List of site IDs that have causal allele. If None, causal site IDs will be
+        chosen randomly according to `num_causal`.
```

Comment:
Need to document somewhere that site IDs will be sorted in the output.

Reply:
It is documented here
https://github.com/tskit-dev/tstrait/pull/124/files#diff-6f429d2ba8d90fd4d6bd4abbfb83ae36dab4e3b660094e94e879c14078af4310R218